docs/api-reference/webgl/framebuffer.md

```diff
@@ -249,24 +249,30 @@ Clears the contents (pixels) of the framebuffer attachments.
 
 ### readPixels
 
-App can provide pixelArray or have it auto allocated by this method
-    x = 0,
-    y = 0,
-    width,
-    height,
-    format = GL.RGBA,
-    type,
-    pixelArray = null
-
-* Readpixels can be slow as it requires a roundtrip to the GPU
+Supports, two modes of operation, `Synchronous` and `Asynchronous`. In `Synchronous` mode copies data into an array Object, in `Asynchronous` mode copies data into a Buffer object. `Asynchronous` mode is supported only when `WebGL2` context is used, when using `WebGL`, it falls back to `Synchronous` mode.
```

Comment:
I will update this documentation to include new `readPixelsToBuffer` before landing.